types/node-fetch/externals.d.ts

```diff
@@ -17,5 +17,5 @@ export interface AbortSignal {
 
     dispatchEvent: (event: any) => boolean;
 
-    onabort?: null | ((this: AbortSignal, event: any) => void);
+    onabort: ((this: AbortSignal, ev: Event) => any) | null;
```

Comment:
I believe this is testable. In the test file, reference `lib.dom.ts` at the top and then add a new test passing in the global `AbortSignal` to node-fetch.

Reply:
Hi @southpolesteve , I am closing this MR as I found a way around

Reply:
@milind12 What is the way around, please? I'm still facing the same issue (https://github.com/node-fetch/node-fetch/issues/741) which your PR might solve.

Reply:
@ttrepka please check there is one http option to set the timeout, (not a recommended way) but it has always worked for me. https://www.npmjs.com/package/node-fetch#options

Reply:
Thanks, I would rather use the recommended way. I guess I can do this for now but would be nicer if I didn't:
```
fetch(url, { signal: abortController.signal as any })
```